styles/work-section.css

```diff
@@ -66,3 +67,19 @@
   border-radius: 4px;
   border: 0;
 }
+
+@media only screen and (min-width: 1090px) {
+  .work-lists {
+    grid-template-columns: repeat(3, 1fr);
+    margin: 0 8.9735rem;
+    row-gap: 32px;
+  }
+}
+
+@media only screen and (max-width: 1090px) and (min-width: 768px) {
+  .work-lists {
+    grid-template-columns: repeat(2, 1fr);
+    margin: 0 auto;
+    row-gap: 32px;
+  }
```

Comment:
- [ ] In the figma design, there are six work cards but you have seven. Kindly remove one and also, the horizontal gaps between them should be smaller than you used. I am attaching the images below:
yours: 
![image](https://user-images.githubusercontent.com/91486791/236191580-844690c8-1605-4b7c-b57c-65f95eaf4953.png)
figma: 
![image](https://user-images.githubusercontent.com/91486791/236191946-d8100763-1a7a-4f02-9654-a940e8cd90df.png)